cupyx/jit/_builtin_funcs.py

```diff
@@ -94,14 +96,53 @@ def call(self, env, *args, **kwds):
 class SyncThreads(BuiltinFunc):
 
     def __call__(self):
-        """Calls ``__syncthreads()``
+        """Calls ``__syncthreads()``.
+
+        .. seealso:: `Synchronization functions`_
+
+        .. _Synchronization functions:
+            https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#synchronization-functions
         """
         super.__call__(self)
 
     def call_const(self, env):
         return Data('__syncthreads()', _cuda_types.void)
 
 
+class SyncWarp(BuiltinFunc):
+
+    def __call__(self, mask=0xffffffff):
+        """Calls ``__syncwarp()``.
+
+        Args:
+            mask (int): Active threads in a warp. Default is 0xffffffff.
+
+        .. seealso:: `Synchronization functions`_
+
+        .. _Synchronization functions:
+            https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#synchronization-functions
+        """
+        super().__call__()
+
+    def call(self, env, **kwargs):
```

Comment:
How about writing in this style?
```suggestion
    def call(self, env, *, mask=None):
```